packages/obonode/obojobo-chunks-table/table-options-icon-hovered.svg

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This SVGs have a lot of extra content that could be removed - does Inkscape have a minimal SVG export?